test-configs.yaml

```diff
@@ -1027,11 +1020,76 @@ device_types:
     filters:
       - whitelist: {defconfig: ['defconfig']}
 
+  r8a7743-iwg20d-q7:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    dtb: 'r8a7743-iwg20d-q7-dbcm-ca.dtb'
+    filters:
+      - whitelist: {defconfig: ['multi_v7_defconfig', 'shmobile_defconfig']}
+
+  r8a7745-iwg22d-sodimm:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    dtb: 'r8a7745-iwg22d-sodimm-dbhd-ca.dtb'
+    filters:
+      - whitelist: {defconfig: ['multi_v7_defconfig', 'shmobile_defconfig']}
+
+  r8a77470-iwg23s-sbc:
+    mach: renesas
+    class: arm-dtb
+    boot_method: uboot
+    filters:
+      - whitelist: {defconfig: ['multi_v7_defconfig', 'shmobile_defconfig']}
+
+  r8a774a1-hihope-rzg2m-ex:
+    mach: renesas
+    class: arm64-dtb
+    boot_method: uboot
+    filters:
+      - whitelist: {defconfig: ['defconfig']}
+
+  r8a774a1-hihope-rzg2n-ex:
```

Comment:
Typo: (b1, not a1): `r8a774b1-hihope-rzg2n-ex`

Reply:
Sigh. Thanks!

Reply:
And fixed.